src/Spago/Build.hs

```diff
@@ -168,9 +235,9 @@ repl newPackages sourcePaths pursArgs depsOnly = do
           [ "The package called 'psci-support' needs to be installed for the repl to work properly."
           , "Run `spago install psci-support` to add it to your dependencies."
           ]
-      let globs = Packages.getGlobs deps depsOnly (configSourcePaths <> sourcePaths)
+      let globs = Packages.getGlobs deps depsOnly configSourcePaths <> sourcePaths
```

Comment:
I am not sure why remove the parentheses here since this means a different thing?

Reply:
Oh, good catch, I don't remember why I did that...